From 8f46a843664d658bd3e43d65cac2a6b32a906d42 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 22 Aug 2022 17:00:51 +0800
Subject: [PATCH] Revert "Linux 5.10.137"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 74ded189e5e4df83aaa1478f7a021f904105c8dc.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b3bfdf51232f..4802db88e9d9 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 137
+SUBLEVEL = 136
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

